pontoon/base/models.py

```diff
@@ -935,6 +935,14 @@ def get_details(parts):
                 'approved_strings',
             )
 
+            # Make output JSON serializable
+            for value in values:
+                deadline = value['resource__deadline']
+                if deadline:
+                    value['resource__deadline'] = str(deadline)
```

Comment:
I used `datetime.isoformat()` in Socorro. Not sure if that's what `str` does under the hood, but might as well be explicit, and `isoformat` is the best format for sharing dates between computer systems in my opinion.

Reply:
From my past experiences: `datetime.isoformat` + custom Json encoder to keep that in one place.
e.g.
https://gist.github.com/drmalex07/5149635e6ab807c8b21e